test/functional/sqlsrv/srv_231_string_truncation_text.phpt

```diff
@@ -95,7 +91,7 @@ function getData($k)
     return $data;
 }
 
-function sqlType($k)
+function getFieldType($k)
```

Comment:
perhaps change name to getSQLType?

Reply:
no, there's a helper method with this name
I changed it to SQLType() instead